src/cromshell/utilities/CromshellConfig.py

```diff
@@ -0,0 +1,117 @@
+import logging
+from pathlib import Path
+
+
+class CromshellConfig:
+    """Setup Cromshell config details"""
+
+    @classmethod
+    def user_defined_slim_metadata_parameters(cls, slim_metadata_parameters_user):
+        """Override Cromwell Slim Metadata From Command Line"""
```

Comment:
"Metadata query keys", as we don't override data but override which keys we send to the server for querying.